content/en/docs/release-notes/rn-armory-halyard/armory-halyard-v1-9-0.md

```diff
@@ -8,25 +8,24 @@ aliases:
 
 ## 04/23/2020 Release Notes
 
-This version is required to deploy Armory Spinnaker 2.19+. 
+This version is required to deploy Armory 2.19+. 
 
 
 ## Known Issue
 
 ### Halyard version
 
-There is a known issue where Armory Halyard 1.9.0 fails to install Armory Spinnaker 2.18.1. The Pod for Echo enters a crash loop.
+There is a known issue where Armory extended Halyard 1.9.0 fails to install Armory 2.18.1. The Pod for Echo enters a crash loop.
```

Comment:
Armory-extended Halyard

Reply:
> I got tired of highlighting "Armory extended Halyard"
> 
> RE the Operator. I think we should standardize on Armory Operator unless we are specifically talking about the open source version, which maybe we should call "Operator for Spinnaker" ?

We can probably safe call the oss version `Spinnaker Operator`. The LF would likely own any trademarks/copyrights for "Operator" in this context (if they exist). As a legal entity, the Spinnaker project exists under the LF umbrella, so it's using its own trademark.  There's also an established pattern of Kubernetes operators being called XYZ Operator.